sklearn/_build_utils/__init__.py

```diff
@@ -6,15 +6,8 @@
 
 from __future__ import division, print_function, absolute_import
 
-HASH_FILE = 'cythonize.dat'
 DEFAULT_ROOT = 'sklearn'
 
-# WindowsError is not defined on unix systems
-try:
-    WindowsError
-except NameError:
-    WindowsError = None
```

Comment:
No need, already done in the other PR.

On 18/02/2016 17:18, Tom Dupré la Tour wrote:
>
> In sklearn/_build_utils/__init__.py 
> <https://github.com/scikit-learn/scikit-learn/pull/6396#discussion_r53337567>:
>
> >  DEFAULT_ROOT = 'sklearn'
> >
> > -# WindowsError is not defined on unix systems
> > -try:
> > -    WindowsError
> > -except NameError:
> > -    WindowsError = None
>
> @ogrisel <https://github.com/ogrisel>
>
>     You can clean up the useless WindowsError and cython.dat in
>     sklearn//build_utils/__init_/.py while you are at it.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/scikit-learn/scikit-learn/pull/6396/files#r53337567>.
>

-- 
Arthur Mensch